Dockerfile

```diff
@@ -0,0 +1,29 @@
+#Deriving the latest base image
+FROM python:latest
```

Comment:
Can we pin to a specific python version instead of latest? 3.9, 3.10?

Reply:
done